### ISSUE TITLE
[0.4.3] Bad Material Stonework Factory recipe

### ISSUE THREAD

**Issue by mouseas** [NillerMedDild/Enigmatica6#1785]
**Modpack version**
0.4.3 via CurseForge/Overwolf

**Describe the bug**
The Material Stonework Factory process for making glass from cobblestone is producing white stained glass, because "Action: Crushing" on Gravel produces White Sand instead of Sand. I would expect White Sandstone to Crush into White Sand, but not Gravel. I tried using regular Sandstone with "Action: Crushing" but it doesn't do anything, and White Sandstone doesn't Crush into White Sand. I also went through the whole MSF recipe list in JEI and didn't find regular sand or glass in it.

So tl;dr please change MSF recipes to:

- Gravel &rarr; Crush &rarr; Sand
- [Color] Sandstone &rarr; Crush &rarr; [Color] Sand

As an aside, there are too many materials in the MSF's starting block types for a one-direction click-to-cycle input to be viable any more. If you click it by accident you have to cycle through the whole list to get back to, say, cobblestone. I know that's Industrial Foregoings creator's responsibility, can you maybe pass it on?

**Logs**
n/a

**Screenshots**
![Screenshot 2021-03-22 173212](https://user-images.githubusercontent.com/1171633/112071017-96d47b80-8b34-11eb-8c21-66e52d039e77.png)
![Screenshot 2021-03-22 173120](https://user-images.githubusercontent.com/1171633/112071019-976d1200-8b34-11eb-8c62-a2b94ae2963b.png)

**Comment by OneLemonyBoi** [comment]
AFAIK this is intended by the Mod Author, not sure about the button part though.

**Comment by NillerMedDild** [comment]
Button part requested here #1420 :)
Will look into the other issue tonight.
